### PR TITLE
Fix setup.py; replace tsne with sklearn.manifold.TSNE

### PR DESCRIPTION
**Pull request by fanli-gcb**
In `setup.py`:
- explicitly specify `packages=['wishbone']`
- remove `tsne` as a dependency

In `wb.py`:
- remove import of `tsne`
- use sklearn.manifold.TSNE implementation (as suggested by https://github.com/danielfrg/tsne)

### CODE REVIEW

**Comment by ManuSetty** [comment]
Thanks a lot for this update! I will incorporate this in the next release.

**Comment by ManuSetty** [comment]
I have held off switching over to Sklearn tSNE since the results from two versions of tSNE seem to be inconsistent. I will explore this further and pull this in.

**Comment by fanli-gcb** [comment]
Part of the inconsistency may be due to seeding of the RNG. As far as I can tell, the python implementation at https://github.com/osdf/py_bh_tsne/tree/master/fasttsne does not explicitly set a seed. The sklearn implementation has an option to do so, but it's sort of irrelevant because seeding doesn't appear to work identically using `numpy.random` versus `srand`.

**Comment by ManuSetty** [comment]
In our testing, tSNE from he package `bhtsne` gives better results compared to the sklearn version. The dependency has been updated to use `bhtsne` instead of `bh_sne`.
